nuget/Animations.nuspec

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+   <metadata minClientVersion="2.8.1">
+       <id>FormsCommunityToolkit.Animations</id>
+       <version>$version$</version>
+       <title>Xamarin.Forms Community Toolkit: Animations</title>
+       <authors>FormsCommunityToolkit</authors>
+       <owners>FormsCommunityToolkit</owners>
+       <projectUrl>https://github.com/FormsCommunityToolkit/FormsCommunityToolkit</projectUrl>
+       <licenseUrl>https://github.com/FormsCommunityToolkit/FormsCommunityToolkit/blob/master/LICENSE</licenseUrl>
+       <iconUrl>https://github.com/FormsCommunityToolkit/FormsCommunityToolkit/blob/master/Media/OrganonLogo.64x64.png</iconUrl>
+       <requireLicenseAcceptance>false</requireLicenseAcceptance>
+       <description>
+         Xamarin.Forms Community Toolkit Animations Library
+         Built against: 2.3.2.127
```

Comment:
I see another version in the samples?

Reply:
Yes, you are right, the 2.3.4.247 version.